backend/internal/service/handler/reviews/get_media_reviews.go

```diff
@@ -3,34 +3,22 @@ package reviews
 import (
 	"fmt"
 	"platnm/internal/models"
+	"strconv"
 
 	"github.com/gofiber/fiber/v2"
 )
 
-type PaginationRequest struct {
```

Comment:
changed this endpoint because it was a GET that required a body to be passed in, which actually is not standard! (axios doesn't let u pass body params for GETs so we weren't able to use the endpoint as-is)

also made it so that default values are provided if u don't pass anything in